Update Feature comparison of versions

hamidra opened the pull request:
there is a confusion about how the plans are compared for non admin accounts. The only plan that protects non admin account is Azure Multifactor Authentication

azurecla commented:
Hi __@joseidz__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (joseidz). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

rmca14 commented:
Sent to Gene for approval.

hamidra committed:
Update Feature comparison of versions

there is a confusion about how the plans are compared for non admin accounts. The only plan that protects non admin account is Azure Multifactor Authentication
